.travis.yml

```diff
@@ -18,7 +18,7 @@ install:
 - pip install coveralls
 - pip install pytest-cov
 before_script:
-- wget "http://chromedriver.storage.googleapis.com/81.0.4044.20/chromedriver_linux64.zip"
+- wget "http://chromedriver.storage.googleapis.com/83.0.4103.39/chromedriver_linux64.zip"
```

Comment:
Would it be worth it to call https://chromedriver.storage.googleapis.com/LATEST_RELEASE to get the latest stable version dynamically?

Reply:
Good point, I always wished that a url like https://chromedriver.storage.googleapis.com/LATEST_RELEASE/chromedriver_linux64.zip would exist but with that link I can just dynamically pull the build number. Ill give it a shot.

Reply:
```
- LATEST_CHROMEDRIVER_VERSION=`curl -s "https://chromedriver.storage.googleapis.com/LATEST_RELEASE"`
- wget "http://chromedriver.storage.googleapis.com/${LATEST_CHROMEDRIVER_VERSION}/chromedriver_linux64.zip"
```

Seems to do the trick, thanks @neiljhowell